### ISSUE TITLE
LCP parsing issue

### ISSUE THREAD

**Issue by kznsq** [Beep6581/RawTherapee#4137]
Hi! The LCP parser cannot parse some LCP files, because they contain a 'PerspectiveModel' section in the XML attributes.
For example, profile "Adobe (Apple iPhone 7 back camera 3.99mm f/1.8)":

      <stCamera:PerspectiveModel
       stCamera:Version="2"
       stCamera:FocalLengthX="0.832288"
       stCamera:FocalLengthY="0.832288"
       stCamera:ImageXCenter="0.495976"
       stCamera:ImageYCenter="0.480453"
       stCamera:ResidualMeanError="0.000636"
       stCamera:ResidualStandardDeviation="0.001126"
       stCamera:RadialDistortParam1="0.065816"
       stCamera:RadialDistortParam2="-0.180165"
       stCamera:RadialDistortParam3="0.208636"/>
      </rdf:Description>

**Comment by TooWaBoo** [comment]
Modify the LCP like below. I don't know, why Adobe uses so many different structures for their LCPs.

```
    <stCamera:PerspectiveModel>
      <rdf:Description
        stCamera:Version="2"
        stCamera:FocalLengthX="0.832288"
        stCamera:FocalLengthY="0.832288"
        stCamera:ImageXCenter="0.495976"
        stCamera:ImageYCenter="0.480453"
        stCamera:ResidualMeanError="0.000636"
        stCamera:ResidualStandardDeviation="0.001126"
        stCamera:RadialDistortParam1="0.065816"
        stCamera:RadialDistortParam2="-0.180165"
        stCamera:RadialDistortParam3="0.208636">
      </rdf:Description>
    </stCamera:PerspectiveModel>
  </rdf:Description>
</rdf:li>
```

**Comment by kznsq** [comment]
I made some changes and it is seems to be ok:

    if (!strcmp("PerspectiveModel", src)) {
        pProf->firstLIDone = true;
        pProf->inPerspect = true;
        parseAttr = true;
        //return;
    } else if (!strcmp("FisheyeModel", src)) {
        pProf->firstLIDone = true;
        pProf->inPerspect = true;
        pProf->_profile->isFisheye = true; // just misses third param, and different path, rest is the same
        parseAttr = true;
        //return;
    } else if (!strcmp("Description", src)) {
        parseAttr = true;
    }

**Comment by Beep6581** [comment]
@kznsq can you post that in the form of a `git diff`?

**Comment by Beep6581** [comment]
@agriggio what do we do with this?

**Comment by agriggio** [comment]
I'll take a look

**Comment by agriggio** [comment]
fixed in 4d81812d137213a6b71bed64aa5aa57601960246
